src/globals.h

```diff
@@ -44,6 +44,8 @@ extern global_ctx_t G_context;
  */
 extern token_data_ctx_t G_token_symbols;
 
+extern create_token_ctx_t G_create_token_data;
```

Comment:
The naming does not make sense now.
This context is made to be used by the new command to allow create token transactions, namely to allow data outputs when creating NFTs.